commands/inviteLink.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
 	helpMsg: true,
 	execute(message) {
 		const {inviteLink} = message.client.config;
+		const {botSpam} = message.client.config.channelIDs;
```

Comment:
can be changed to 
`const {inviteLink, channelIDs} = message.client.config;`

To get the botSpam channel you just do:
`message.client.channels.get(channelIDs,botSpam)...`

Reply:
feedback acted on :^)